torchbox/torchbox.com
# Add default social sharing images and use webp format for all social images

No ticket - spotted when testing social sharing that there were a lot of pages without a social image set

### Description of Changes Made

If there is no social image found for a page, use a generic torchbox image, coloured according to the theme of the page. Includes the earth images for future-proofing.

Use webp for all social sharing images as all major platforms support his.

### How to Test

In your local build, visit a service page or the home page and make sure there is no social image set (some page types such as work pages automatically use the main image on the page). Inspect the `<head>` element, and check that the meta tags are set for `og:image`, `og:image:width`, `og:image:height` and `twitter:image` (note that the twitter image uses a different width and height to the og image).

Edit the page theme - the image used should update to reflect the theme. If no theme is set, the coral image will be used.

### Screenshots

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
